src/plugins/intel_cpu/src/emitters/tpp/common/kernel_executors/brgemm.hpp

```diff
@@ -0,0 +1,129 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "common/utils.hpp"
+#include "emitters/snippets/brgemm_base.hpp"
+#include "emitters/utils.hpp"
+#include "libxsmm.h"
+
+namespace ov::intel_cpu::tpp {
+
+struct BrgemmKernelConfig : public BrgemmGenericKernelConfig {
+public:
+    BrgemmKernelConfig(const element::Type& in0_dtype, const element::Type& in1_dtype);
+    BrgemmKernelConfig() = delete;
+
+    std::unique_ptr<snippets::KernelExecutorBase::GenericConfig> get_clone_ptr() const override {
+        return std::unique_ptr<BrgemmKernelConfig>(new BrgemmKernelConfig(*this));
+    }
+
+    bool operator==(const BrgemmKernelConfig& rhs) const;
+    bool operator!=(const BrgemmKernelConfig& rhs) const {
+        return !(*this == rhs);
+    }
+
+    void update(int64_t M, int64_t N, int64_t K, int64_t LDA, int64_t LDB, int64_t LDC, float beta);
+
+    size_t hash() const override {
+        return m_hash;
+    }
+    size_t compute_hash() const;
+
+    libxsmm_bitfield get_static_compile_flags() const {
+        return m_static_params->m_compile_flags;
+    }
+    libxsmm_bitfield get_compile_flags() const {
+        return m_compile_flags;
+    }
+    void set_compile_flags(bool zero_beta) {
```

Comment:
This method can only set `LIBXSMM_GEMM_FLAG_BETA_0` flag. Could you please reflect it in the name? `set_compile_flags` is too common from my perspective

Reply:
The name is updated, thanks Vladislav!